pkg/api/v1/types.go

```diff
@@ -1799,6 +1801,20 @@ type ComponentStatusList struct {
 	Items []ComponentStatus `json:"items" description:"list of component status objects"`
 }
 
+// MetadataVolumeSource represents an host directory mounted on a POD. It stores pod's metadata info
+type MetadataVolumeSource struct {
+	// Items is a list of metadata file name
+	Items []MetadataFile `json:"items",omitempty" description:"list of metadata file"`
+}
+
+// MetadataFile represents information to create the file containing the pod field
+type MetadataFile struct {
+	// Required: Name is the name of the file
+	Name string `json:"name,omitempty" description:"Name contains the name of the file to created"`
```

Comment:
Description should be "the name of the file to be created".  Notice, descriptions shouldn't include the actual name of the field and should begin with lowercase.